heureka/ui/src/components/issues/IssuesList.jsx

```diff
@@ -18,12 +18,15 @@ const IssuesList = ({ items, isLoading }) => {
   return (
     <>
       {/* clickableTable Table allow changes the background by css when hovering or active*/}
-      <DataGrid columns={10} className="clickableTable">
+      <DataGrid
+        gridColumnTemplate="2fr 2fr 4fr 2fr 2fr 2fr 2fr 2fr 2fr 2fr"
```

Comment:
is it really necessary to manually set the third column to double the size? If it has more content than the other columns then the auto layout should pretty much take care of it already.

Reply:
```suggestion
        columns={10}
```